processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:05:24.580209905 -0500
+++ memcpu.hex	2019-02-22 09:05:24.527208331 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.sv (4 of 8)
--- memsim.hex	2019-02-22 09:05:25.959250889 -0500
+++ memcpu.hex	2019-02-22 09:05:25.906249315 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 9)
processing file asmFiles/unitTest_control_hazard.asm (2 of 9)
processing file asmFiles/unitTest_dumb.sv (3 of 9)
--- memsim.hex	2019-02-22 09:09:29.436486743 -0500
+++ memcpu.hex	2019-02-22 09:09:29.380485077 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 9)
--- memsim.hex	2019-02-22 09:09:30.828528113 -0500
+++ memcpu.hex	2019-02-22 09:09:30.736525379 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.sv (5 of 9)
--- memsim.hex	2019-02-22 09:09:32.198568829 -0500
+++ memcpu.hex	2019-02-22 09:09:32.155567552 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (6 of 9)
processing file asmFiles/unitTest_jumpInstructions.asm (7 of 9)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (8 of 9)
processing file asmFiles/unitTest_raw_hazard_solv.asm (9 of 9)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
--- memsim.hex	2019-02-22 09:37:12.307906139 -0500
+++ memcpu.hex	2019-02-22 09:10:10.301701213 -0500
@@ -1,12 +1,11 @@
-:0400000034080002BE
-:0400010034090002BC
-:0400020011090004DC
-:0400030008000006EB
-:0400040034080008B4
-:0400050008000006E9
-:04000600FFFFFFFFFA
-:0400070034080003B6
-:0400080034090002B5
-:040009001509FFFADC
-:04000A0008000006E4
+:040000003442008006
+:040001008C4300002C
+:040002008C44000426
+:04000300006428214C
+:04000400AC450008FF
+:04000500004810217E
+:0400060001E1782379
+:04000700FFFFFFFFF9
+:040020008C4300000D
+:0400210000000001DA
 :00000001FF
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
--- memsim.hex	2019-02-22 09:37:12.641916066 -0500
+++ memcpu.hex	2019-02-22 09:10:10.301701213 -0500
@@ -1,6 +1,11 @@
-:04000000340A000AB4
-:04000100340B000AB2
-:04000200114B00019D
-:04000300340A0002B9
-:04000400FFFFFFFFFC
+:040000003442008006
+:040001008C4300002C
+:040002008C44000426
+:04000300006428214C
+:04000400AC450008FF
+:04000500004810217E
+:0400060001E1782379
+:04000700FFFFFFFFF9
+:040020008C4300000D
+:0400210000000001DA
 :00000001FF
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:37:13.092929470 -0500
+++ memcpu.hex	2019-02-22 09:10:10.301701213 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:37:13.705947688 -0500
+++ memcpu.hex	2019-02-22 09:10:10.301701213 -0500
@@ -6,6 +6,6 @@
 :04000500004810217E
 :0400060001E1782379
 :04000700FFFFFFFFF9
+:040020008C4300000D
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:37:46.924934947 -0500
+++ memcpu.hex	2019-02-22 09:37:46.816931737 -0500
@@ -7,5 +7,5 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
+:0400220000000004D6
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:37:48.405978963 -0500
+++ memcpu.hex	2019-02-22 09:37:48.294975664 -0500
@@ -7,5 +7,5 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
+:0400220000000004D6
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:40:52.180441317 -0500
+++ memcpu.hex	2019-02-22 09:40:52.081438374 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:40:53.736487572 -0500
+++ memcpu.hex	2019-02-22 09:40:53.617484035 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:41:37.059775420 -0500
+++ memcpu.hex	2019-02-22 09:41:36.969772744 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:41:38.435816325 -0500
+++ memcpu.hex	2019-02-22 09:41:38.338813441 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:56:37.734548971 -0500
+++ memcpu.hex	2019-02-22 09:56:37.637546087 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:56:39.122590232 -0500
+++ memcpu.hex	2019-02-22 09:56:39.055588240 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 09:56:52.278981316 -0500
+++ memcpu.hex	2019-02-22 09:56:52.237980096 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 09:56:53.766025519 -0500
+++ memcpu.hex	2019-02-22 09:56:53.603020674 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2019-02-22 10:52:45.074643255 -0500
+++ memcpu.hex	2019-02-22 10:52:44.997640966 -0500
@@ -13,6 +13,7 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040020000000BA5CC6
+:040021000000BA5CC5
+:0400260000000018BE
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 10:53:01.712138001 -0500
+++ memcpu.hex	2019-02-22 10:53:01.662136514 -0500
@@ -7,5 +7,5 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
+:040022008C4300000B
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 10:53:03.116179752 -0500
+++ memcpu.hex	2019-02-22 10:53:03.058178028 -0500
@@ -7,5 +7,5 @@
 :04000600FFFFFFFFFA
 :0400200000000002DA
 :0400210000000003D8
-:0400220000000002D8
+:0400220034420080E4
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 10:53:56.147756736 -0500
+++ memcpu.hex	2019-02-22 10:53:56.083754833 -0500
@@ -7,5 +7,4 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 10:53:57.344792332 -0500
+++ memcpu.hex	2019-02-22 10:53:57.280790414 -0500
@@ -7,5 +7,4 @@
 :04000600FFFFFFFFFA
 :0400200000000002DA
 :0400210000000003D8
-:0400220000000002D8
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_forward_solv2.asm (1 of 1)
--- memsim.hex	2019-02-22 10:55:12.142016546 -0500
+++ memcpu.hex	2019-02-22 10:55:12.085014850 -0500
@@ -7,5 +7,4 @@
 :04000600FFFFFFFFFA
 :0400200000000002DA
 :0400210000000003D8
-:0400220000000002D8
 :00000001FF
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
--- memsim.hex	2019-02-22 10:58:27.757833439 -0500
+++ memcpu.hex	2019-02-22 10:58:27.688831387 -0500
@@ -7,5 +7,5 @@
 :0400060001E1782379
 :04000700FFFFFFFFF9
 :0400210000000001DA
-:0400220000000001D9
+:040022008C4300000B
 :00000001FF
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
--- memsim.hex	2019-02-22 10:58:29.005870551 -0500
+++ memcpu.hex	2019-02-22 10:58:28.957869140 -0500
@@ -7,5 +7,5 @@
 :04000600FFFFFFFFFA
 :0400200000000002DA
 :0400210000000003D8
-:0400220000000002D8
+:0400220034420080E4
 :00000001FF
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
processing file asmFiles/unitTest_branchInstructions.asm (1 of 8)
processing file asmFiles/unitTest_control_hazard.asm (2 of 8)
processing file asmFiles/unitTest_dumb.sv (3 of 8)
processing file asmFiles/unitTest_forward_solv2.asm (4 of 8)
processing file asmFiles/unitTest_halt.asm (5 of 8)
processing file asmFiles/unitTest_jumpInstructions.asm (6 of 8)
processing file asmFiles/unitTest_raw_hazard_nsolv.asm (7 of 8)
processing file asmFiles/unitTest_raw_hazard_solv.asm (8 of 8)
